testdata/scripts/nodes/cosmos/list/list.txtar

```diff
@@ -1,3 +1,5 @@
+env CL_COSMOS_CMD=chainlink-cosmos
```

Comment:
Empty or unset env vars used to signal that we should run in-memory, but they are no longer meaningful. We could switch to using this value as the default in those cases, which would allow us to drop this boilerplate and re-parallelize the tests